src/Pages/Dashboard/Dashboard.jsx

```diff
@@ -61,7 +61,7 @@ export const Dashboard = () => {
   },[])
 
   return (
-    <div className='flex'>
+    <div className='flex sidebar_position'>
```

Comment:
you can replace this class with this: 'flex-l di' because in the dash.css, you're using the sidebar_position before 60em, so you can replace this with 'di' and 'flex-l' is because the display:flex is going active after 60em.
`<div className='di flex-l'>`